rllib/utils/exploration/tests/test_random_encoder.py

```diff
@@ -0,0 +1,67 @@
+import sys
```

Comment:
This is great! Could we actually activate this test? By adding it to `rllib/BUILD` to the other `utils/exploration` tests?
This will make sure this feature never breaks in the future.